packages/select/src/FilterMultiSelect/components/Root/Root.tsx

```diff
@@ -22,7 +22,9 @@ export interface RootProps
 }
 
 interface MenuTriggerProps {
+  isOpen?: boolean
   defaultOpen?: boolean
+  onOpenChange?: (isOpen: boolean) => void
```

Comment:
Suggestion: This interface could be exported and extended by `MenuTriggerProviderProps` for consistency (eg. if one of the prop's type changes)